numpy/f2py/auxfuncs.py

```diff
@@ -3,8 +3,8 @@
 
 Auxiliary functions for f2py2e.
 
-Copyright 1999,2000 Pearu Peterson all rights reserved,
-Pearu Peterson <pearu@ioc.ee>
+Copyright 1999 - 2011 Pearu Peterson all rights reserved.
+Copyright 2011 - 2023 NumPy Developers.
```

Comment:
Do we have precedent for adding another copyright line to modified files? I think this is covered by the overall project license, but if you feel strongly then perhaps we should simply remove the dates altogether, as suggested [here](https://www.nexb.com/do-you-really-need-to-update-the-copyright-each-new-year/) to prevent additional yearly maintenance burden.

Reply:
I don't think we can / should remove Pearu's original copyright line, but at the same time I wanted to emphasize the current state of maintenance. Does 2011--present sound alright?